frontend/ui/widget/configdialog.lua

```diff
@@ -61,11 +62,19 @@ function OptionTextItem:init()
         }
     else
         self.active_key_events = {
-            Select = { {"Press"}, doc = "chose selected item" },
+            --Select = { {"Press"}, doc = "chose selected item" },
```

Comment:
Comment or remove? For now if you're changing the line you might as well fix the typo from `chose` to `choose` in the process ;-)

Reply:
Hum, I'm still in the process of deciding, because I think this whole process of setting active_key_events when onFocus is not used anymore and does nothing.